pkg/corerp/frontend/controller/environments/getrecipemetadata.go

```diff
@@ -101,7 +101,7 @@ func (r *GetRecipeMetadata) Run(ctx context.Context, w http.ResponseWriter, req
 	return rest.NewOKResponse(versioned), nil
 }
 
-func (r *GetRecipeMetadata) GetRecipeMetadataFromRegistry(ctx context.Context, recipeProperties datamodel.EnvironmentRecipeProperties, recipeDataModel *datamodel.Recipe) (recipeParameters map[string]any, err error) {
+func (r *GetRecipeMetadata) GetRecipeMetadataFromRegistry(ctx context.Context, recipeProperties datamodel.EnvironmentRecipeProperties, recipeDataModel *datamodel.Recipe, envID string) (recipeParameters map[string]any, err error) {
```

Comment:
Need comments for function definition

Reply:
added comments

Reply:
nit: for this function, is it more accurate to say '.. for a given recipe name and template path' or ' for given recipe properties  and datamodel' ?